### PR TITLE
Add certificate check KB.

### PR DESCRIPTION
**Pull request by nmasdoufi-ol**
Add certificate check KB.

### CODE REVIEW

File: WEB_SERVICE/WEB/_HIGH/CERTIFICATE_CHECK/description.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+1. **Expiration Management:** An expired certificate can no longer be trusted and may lead to security warnings or connection failures.
+
+2. **Renewal Alerts:** Certificates approaching their expiration date (within 30 days) should be renewed promptly to avoid service interruptions.
+
+3. **Key Strength:** Certificates that use RSA keys shorter than 2048 bits are considered weak and vulnerable to brute-force attacks.
+
+4. **Signature Algorithm Integrity:** Outdated algorithms like MD5 or SHA1 are vulnerable to collision attacks and should not be used.
+
+5. **Hostname Verification:** The certificate's Common Name (CN) or Subject Alternative Name (SAN) should match the target hostname to prevent potential man-in-the-middle attacks.
```
Comment (elyousfi5) [inline WEB_SERVICE/WEB/_HIGH/CERTIFICATE_CHECK/description.md:13]: in here you are giving remediation actions which should be in recommendation. The description file should have just the definition of that vulnerability class and the impact of it.

File: WEB_SERVICE/WEB/_HIGH/CERTIFICATE_CHECK/meta.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      "CC_6_7",
+      "CC_7_1",
+      "CC_7_2"
+    ],
+    "CCPA": [
+      "CCPA_1798_150"
+    ],
+    "GDPR": [
+      "ART_32"
+    ]
```
Comment (elyousfi5) [inline WEB_SERVICE/WEB/_HIGH/CERTIFICATE_CHECK/meta.json:22]: I think you missing some of PCI

**Review by PiranhaSa — COMMENTED** [review]
Can you please change the title because certificate check doesn't reflect the security issue , maybe Failed_....

File: WEB_SERVICE/WEB/_HIGH/CERTIFICATE_CHECK/description.md (hunk 1 above)
Comment (PiranhaSa) [reply WEB_SERVICE/WEB/_HIGH/CERTIFICATE_CHECK/description.md:13]: this is not remediation it's description of certificate check .

**Review by ybadaoui-ostorlab — DISMISSED** [review]

File: WEB_SERVICE/WEB/_HIGH/CERTIFICATE_CHECK/description.md (hunk 1 above)
Comment (nmasdoufi-ol) [reply WEB_SERVICE/WEB/_HIGH/CERTIFICATE_CHECK/description.md:13]: I'm not giving remediation actions in there.

File: WEB_SERVICE/WEB/_HIGH/CERTIFICATE_CHECK/meta.json (hunk 2 above)
Comment (nmasdoufi-ol) [reply WEB_SERVICE/WEB/_HIGH/CERTIFICATE_CHECK/meta.json:22]: I'll check that.

**Review by 3asm — APPROVED** [review]

**Pull request merged**